modules/appeals_api/app/swagger/decision_reviews/v2/swagger.json

```diff
@@ -3421,11 +3422,11 @@
                       "status": "pending",
                       "code": null,
                       "detail": null,
-                      "appealType": "NoticeOfDisagreement",
-                      "appealId": "00000000-1111-2222-3333-444444444444",
                       "location": "http://some.fakesite.com/path/uuid",
-                      "createdAt": "2020-01-02T03:04:05.067Z",
-                      "updatedAt": "2020-01-02T03:04:05.067Z"
+                      "appealId": "00000000-1111-2222-3333-444444444444",
+                      "appealType": "NoticeOfDisagreement",
+                      "createDate": "2020-01-02T03:04:05.067Z",
+                      "updateDate": "2020-01-02T03:04:05.067Z"
```

Comment:
These particular changes aren't related to the updates in this PR, but the fields reordered themselves after I ran the  rake task that regenerates the documentation. These changes don't harm anything and accurately reflect the order of fields returned in the response.